### ISSUE TITLE
Unable to install Ice on Mojave (Xcode 11.1)

### ISSUE THREAD

**Issue by amyspark** [jakeheis/Ice#36]
Hey,

I've tried to install Ice on Mojave with Xcode 11.1 (didn't update to 11.2 because of certain bugs). I got the following:
```
❯ mint install jakeheis/Ice
🌱  Finding latest version of Ice
🌱  Cloning Ice 0.8.0
🌱  Resolving package
🌱  Building package
[1/5] Compiling SwiftCLI ArgumentList.swift
/private/var/folders/l1/b3htrbqd3w3f32x9lk1yll6h0000gn/T/mint/github.com_jakeheis_Ice/.build/checkouts/SwiftCLI/Sources/SwiftCLI/Option.swift:127:21: error: initializer does not override a designated initializer from its superclass
    public override init(_ names: String ..., description: String = "", completion: ShellCompletion = .filename, validation: [Validation<Value>] = []) {
                    ^
/private/var/folders/l1/b3htrbqd3w3f32x9lk1yll6h0000gn/T/mint/github.com_jakeheis_Ice/.build/checkouts/SwiftCLI/Sources/SwiftCLI/Option.swift:114:12: note: potential overridden initializer 'init(_:description:completion:validation:)' here
    public init(_ names: [String], description: String, completion: ShellCompletion, validation: [Validation<Value>] = []) {
           ^
/private/var/folders/l1/b3htrbqd3w3f32x9lk1yll6h0000gn/T/mint/github.com_jakeheis_Ice/.build/checkouts/SwiftCLI/Sources/SwiftCLI/Option.swift:141:21: error: initializer does not override a designated initializer from its superclass
    public override init(_ names: String ..., description: String = "", completion: ShellCompletion = .filename, validation: [Validation<Value>] = []) {
                    ^
/private/var/folders/l1/b3htrbqd3w3f32x9lk1yll6h0000gn/T/mint/github.com_jakeheis_Ice/.build/checkouts/SwiftCLI/Sources/SwiftCLI/Option.swift:114:12: note: potential overridden initializer 'init(_:description:completion:validation:)' here
    public init(_ names: [String], description: String, completion: ShellCompletion, validation: [Validation<Value>] = []) {
           ^
🌱  Encountered error during "swift build -c release -Xswiftc -target -Xswiftc x86_64-apple-macosx10.14". Use --verbose to see full output
🌱  Failed to build Ice 0.8.0 with SPM
```

I don't know why this happens, but I used to get this error too with SwiftCLI directly. Do you know what is going on?

**Comment by timemach1ne** [comment]
same problem here

```
🌱  Building package
/private/var/folders/g1/y66l17253v900g_zzs5lmrzc0000gn/T/mint/github.com_jakeheis_Ice/.build/checkouts/SwiftCLI/Sources/SwiftCLI/Option.swift:127:21: error: initializer does not override a designated initializer from its superclass
    public override init(_ names: String ..., description: String = "", completion: ShellCompletion = .filename, validation: [Validation<Value>] = []) {
                    ^
/private/var/folders/g1/y66l17253v900g_zzs5lmrzc0000gn/T/mint/github.com_jakeheis_Ice/.build/checkouts/SwiftCLI/Sources/SwiftCLI/Option.swift:114:12: note: potential overridden initializer 'init(_:description:completion:validation:)' here
    public init(_ names: [String], description: String, completion: ShellCompletion, validation: [Validation<Value>] = []) {
           ^
/private/var/folders/g1/y66l17253v900g_zzs5lmrzc0000gn/T/mint/github.com_jakeheis_Ice/.build/checkouts/SwiftCLI/Sources/SwiftCLI/Option.swift:141:21: error: initializer does not override a designated initializer from its superclass
    public override init(_ names: String ..., description: String = "", completion: ShellCompletion = .filename, validation: [Validation<Value>] = []) {
                    ^
/private/var/folders/g1/y66l17253v900g_zzs5lmrzc0000gn/T/mint/github.com_jakeheis_Ice/.build/checkouts/SwiftCLI/Sources/SwiftCLI/Option.swift:114:12: note: potential overridden initializer 'init(_:description:completion:validation:)' here
    public init(_ names: [String], description: String, completion: ShellCompletion, validation: [Validation<Value>] = []) {
           ^

🌱  Encountered error during "swift build -c release -Xswiftc -target -Xswiftc x86_64-apple-macosx10.15"
🌱  Failed to build Ice 0.8.0 with SPM
```

**Comment by thehanlinsun** [comment]
Same here

**Comment by huguesleygue** [comment]
Same issue on Catalina

**Comment by jakeheis** [comment]
The issue should be fixed in `0.8.1`. I'll close for now but feel free to reopen if the issue remains!
